internal/rpc/erc721/ethservice.go

```diff
@@ -0,0 +1,81 @@
+package erc721
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/freeverseio/laos-universal-node/internal/platform/blockchain"
+)
+
+type EthService struct {
+	Ethcli blockchain.EthRPCClient
+}
+
+// ChainId returns the chain ID of the ethService as a hexutil.Big.
+// nolint:revive // needs to be named ChainId for EVM compatibility.
+func (b *EthService) ChainId() *hexutil.Big {
```

Comment:
Why are we reimplementing these eth rpc calls? Go-ethereum already provides these call (BlockNumber, ChainId...)